src/control/pbin/pbin.go

```diff
@@ -39,10 +39,20 @@ const (
 	// DaosAdminLogFileEnvVar is the name of the environment variable which
 	// can be set to enable non-ERROR logging in the privileged binary.
 	DaosAdminLogFileEnvVar = "DAOS_ADMIN_LOG_FILE"
+
+	// DaosFWName is the name of the daos_firmware privileged helper.
+	DaosFWName = "daos_firmware"
+
+	// DaosFWLogFileEnvVar is the name of the environment variable that
+	// can be set to enable non-ERROR logging in the daos_firmware binary.
+	DaosFWLogFileEnvVar = "DAOS_FIRMWARE_LOG_FILE"
```

Comment:
can this also be set through the server configuration file like with the admin log file (helper_log_file)

Reply:
Haven't added that, will have a look.

Reply:
Added.